src/test/resources/org/openeo/geodiasFeatureSnippet.json

```diff
@@ -0,0 +1,408 @@
+{
+  "type": "Feature",
+  "id": "41c7293f-029d-59e8-9a9d-2a304a1c86ec",
+  "geometry": {
+    "type": "MultiPolygon",
+    "coordinates": [
+      [
+        [
+          [
+            9.124823,
+            49.491451
+          ],
+          [
+            9.596831,
+            50.986023
+          ],
+          [
+            5.835512,
+            51.400711
+          ],
+          [
+            5.479662,
+            49.904503
+          ],
+          [
+            9.124823,
+            49.491451
+          ]
+        ]
+      ]
+    ]
+  },
+  "properties": {
+    "collection": "Sentinel1",
+    "status": 0,
+    "license": {
+      "licenseId": "unlicensed",
+      "hasToBeSigned": "never",
+      "grantedCountries": null,
+      "grantedOrganizationCountries": null,
+      "grantedFlags": null,
+      "viewService": "public",
+      "signatureQuota": -1,
+      "description": {
+        "shortName": "No license"
+      }
+    },
+    "productIdentifier": "/eodata/Sentinel-1/SAR/GRD/2021/04/11/title.SAFE",
+    "parentIdentifier": null,
+    "title": "title.SAFE",
+    "description": null,
+    "organisationName": "ESA",
+    "startDate": "%startDate%",
+    "completionDate": "2021-04-11T05:42:11.835Z",
+    "productType": "GRD",
+    "processingLevel": "LEVEL1",
+    "platform": "S1B",
+    "instrument": "SAR",
+    "resolution": 10,
+    "sensorMode": "IW",
+    "orbitNumber": 26415,
+    "quicklook": null,
+    "thumbnail": "https://finder.creodias.eu/files/Sentinel-1/SAR/GRD/2021/04/11/title.SAFE/preview/thumbnail.png",
+    "updated": "2021-04-11T07:55:01.780481Z",
+    "published": "2021-04-11T07:55:01.780481Z",
+    "snowCover": 0,
+    "cloudCover": -1,
+    "keywords": [
```

Comment:
long list of keywords can be shortened, it's not relevant for the test, just keep 2 to keep it similar to the real data?

Reply:
List shortened, has no impact on the test indeed.